Constitution.md

```diff
@@ -174,15 +180,16 @@ This Constitution does not follow the default template laid out by YUSU, but doe
 4. Only paid-up Society members who are also full members of YUSU shall be entitled to vote.
 5. The Committee must be democratically elected in a free and fair election.
     1. All Society members must have the chance to question candidates and submit a vote in private and in absentia (where necessary).
-    2. Votes are counted using Single Transferable Vote.
-    3. if desired, YUSU can provide assistance in the running of any election to ensure fairness or to count votes where necessary.
+    2. Votes are counted using Single Transferable Vote, as specified in Appendix B.
+    3. If desired, YUSU can provide assistance in the running of any election to ensure fairness or to count votes where necessary.
 6. If any vacancies occur in the Committee during the academic year, they shall be democratically filled as soon as is convenient via an EGM (Extraordinary General Meeting).
 7. Members can stand jointly for election to any Committee position in teams of 2 individuals, under the following regulations:
     1. Both members must be paid-up Society members as per 10.1.
     2. If one member of a jointly elected committee position resigns or is dismissed from their role, an EGM (as per 10.6) shall be held to elect a replacement for the committee position. The remaining member may continue their duties if necessary until the by-election, at which point they must stand for reelection if they wish to continue.
     3. In committee-only votes, the members which hold a joint position shall only have one collective vote.
     4. If there is an irreconcilable disagreement between members holding a joint position in regards to a major decision related to their role, there shall be a committee vote in the next committee meeting. All present members shall be allowed to cast a vote on what action shall be taken in order to solve the disagreement and decide on the action to be taken.
 8. Before taking office, Society Committee members must sign a public declaration agreeing to abide by, enforce, and operate in accordance with this Constitution, YUSU's Constitution, YUSU's Policies, and guidelines in the Resource Hub.
+    1. The template for this declaration is given in Appendix A.
```

Comment:
one other thing: is the idea here to `mv Declaration.md appendices/A-declaration.md`? if we're going to continue to have `Roles.md` gitignored and created from `Roles.template.md`, that will require some effort to adjust the build process to put the names from `Roles.md` immediately after appendix A.